app/src/main/java/com/example/weathertimeapp/mvp/presenter/WeatherTimePresenter.kt

```diff
@@ -23,12 +24,11 @@ class WeatherTimePresenter(
         view.setSoftInputMode()
     }
 
-    private fun initRecyclerView() {
-        view.showRecyclerView()
-    }
-
-    private fun createAndShowData() {
-        view.showDataSet(model.createDataSet())
+    private fun requestCityForecast() {
+        model.getForecastByCityId(3429439).subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
```

Comment:
is more readable if you use  a new line on every operator

Reply:
It's true, my bad, changed